templates/zerver/app/left_sidebar.html

```diff
@@ -7,6 +7,7 @@
                     <span class="filter-icon">
                         <i class="fa fa-home" aria-hidden="true"></i>
                     </span>
+                    {#- squash whitespace -#}
```

Comment:
why do we need these whitespace squashes?

Reply:
The whitespace takes up pixels, which makes CSS arithmetic really hard to reason about.